src/modules/components/transaction/transactionDataListItem/transactionDataListItemStructure/transactionDataListItemStructure.tsx

```diff
@@ -104,7 +104,7 @@ export const TransactionDataListItemStructure: React.FC<ITransactionDataListItem
                     </div>
                 </div>
 
-                <div className="flex flex-col items-end gap-y-0.5">
+                <div className="flex shrink-0 flex-col items-end gap-y-0.5">
```

Comment:
I would add truncation here in case the token symbol is very long (for scam tokens for instance)